corehq/apps/geospatial/dispatchers.py

```diff
@@ -0,0 +1,17 @@
+from corehq import toggles
+from corehq.apps.reports.dispatcher import (
+    ReportDispatcher,
+    cls_to_view_login_and_domain,
+)
+
+
+class CaseManagementMapDispatcher(ReportDispatcher):
+    prefix = 'geospatial'
+    map_name = 'GEOSPATIAL_MAP'
+
+    @cls_to_view_login_and_domain
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
We will need to add a feature-flag check here.

Reply:
Updated :+1: